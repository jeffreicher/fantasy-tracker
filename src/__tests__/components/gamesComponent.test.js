import React from 'react'
import renderer from 'react-test-renderer'
import GamesComponent from '../../components/gamesComponent'

describe('Games Component test', () => {
  it('checks if Games Component matches snapshot', () => {
    const tree = renderer.create(<GamesComponent />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
