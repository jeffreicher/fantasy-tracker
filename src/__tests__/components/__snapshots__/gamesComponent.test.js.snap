// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Games Component test checks if Games Component matches snapshot 1`] = `
<div
  className="container gamesContainer"
>
  <div
    className="dateContainer"
  >
    <button
      className="btn btn-primary"
      type="button"
    >
      Previous
    </button>
    <div>
      2 / 8 / 2019
    </div>
    <button
      className="btn btn-primary"
      type="button"
    >
      Next
    </button>
  </div>
</div>
`;
